src/SolutionTemplate/UnoSolutionTemplate/UnoQuickStart.Droid/UnoQuickStart.Droid.csproj

```diff
@@ -23,7 +23,7 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
+    <DebugType>portable</DebugType>
```

Comment:
I was impressed with your diagnosis on Gitter. Why is this necessary, though?

Reply:
Thanks ;) It's related to the recent changes with the mono debugger, where sequence points of the mdb format were not handling partial classes properly, as well as lambdas. Portable PDBs do.